packages/saas-ui-select/src/select.tsx

```diff
@@ -116,7 +116,7 @@ export const Select = forwardRef<SelectProps, 'select'>((props, ref) => {
       <MenuButton as={SelectButton} ref={ref} {...buttonProps}>
         {displayValue || placeholder}
       </MenuButton>
-      <MenuList>
+      <MenuList maxH="200" overflow="scroll">
```

Comment:
I think it's a better idea to use 'overflowY' and 'auto' here as now it will show horizontal and vertical scroll bars on windows and perhaps other browsers/os even if the content doesn't overflow.

Reply:
Fixed!